### PR TITLE
CB-7203 Account tag generation API for experiences

### PR DESCRIPTION
**Pull request by doktoric**

### CODE REVIEW

File: template-manager-tag/src/main/java/com/sequenceiq/cloudbreak/tag/DefaultCostTaggingService.java [hunk 1]
```diff
@@ -47,9 +59,8 @@
             LOGGER.debug("The tag template is: {}", entry.getValue());
             String generatedValue = centralTagUpdater.getTagText(preparationObject, entry.getValue());
             LOGGER.debug("The generated tag value is: {}", generatedValue);
-            addAccountTag(result, entry.getKey(), generatedValue, request.getPlatform());
```
Comment (keyki) [inline template-manager-tag/src/main/java/com/sequenceiq/cloudbreak/tag/DefaultCostTaggingService.java:50]: addAccountTag is unused now, right?

Comment (doktoric) [reply template-manager-tag/src/main/java/com/sequenceiq/cloudbreak/tag/DefaultCostTaggingService.java:50]: so this is the extracted method. If you check upper there is a used method

Comment (keyki) [reply template-manager-tag/src/main/java/com/sequenceiq/cloudbreak/tag/DefaultCostTaggingService.java:50]: Right!

**Comment by doktoric** [comment]
http://ci-cloudbreak.eng.hortonworks.com/job/cloudbreak-pull-request-builder-integration-test/9013/

**Pull request merged**